addons/l10n_ae/data/l10n_ae_data.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- set VAT label to show on invoice report -->
+    <record id="base.ae" model="res.country">
+        <field name="vat_label">TRN</field>
+    </record>
+    <record id="gcc_countries_group" model="res.country.group">
+        <field name="name">GCC VAT implementing States</field>
+        <field name="country_ids" eval="[(6,0,[ref('base.ae'),ref('base.sa')])]"/>
```

Comment:
There are more countries than these 2 I thought?

Reply:
you're right, there are <a href="http://www.gcc-sg.org/en-us/AboutGCC/MemberStates/pages/Home.aspx">six</a> of them in total. I have updated the list.